src/main/java/org/snomed/snowstorm/core/data/domain/jobs/ExportConfiguration.java

```diff
@@ -48,6 +48,15 @@ public class ExportConfiguration {
 	@Schema(description = "If refsetIds are included, this indicates that the export will be a refset-only export.")
 	private Set<String> refsetIds;
 
+	@Schema(defaultValue = "PENDING")
+	private String status;
+
+	@Schema
+	private String exportPath;
```

Comment:
@dmcgihtsdo I assume this is the exported file path. Can you rename it to exportFilepath please? Also we need consider that these files will be deleted during system restart. If they haven't been downloaded already then we need ask users to start a new export.

Reply:
Also can you add @Field(type = FieldType.Keyword) for this please? This field is only useful if we store exported file in S3.

Reply:
Thanks @CoderMChu, I have made those changes.